Module.php

```diff
@@ -311,4 +312,18 @@ protected function corePanels()
             'timeline' => ['class' => 'yii\debug\panels\TimelinePanel']
         ];
     }
+
+    /**
+     * @inheritdoc
+     * @since 2.0.7
+     */
+    protected function defaultVersion()
+    {
+        $packageInfo = Json::decode(file_get_contents(__DIR__ . DIRECTORY_SEPARATOR . 'composer.json'));
+        $extensionName = $packageInfo['name'];
```

Comment:
I prefer more universal solution: it can be re-used as serves as a living example of possible implementation for developers.